### PR TITLE
Solved fix problem with dist css & js files

### PR DESCRIPTION
**Pull request by Manuss20**

### CODE REVIEW

File: minecraftbot/main.ts [hunk 1]
```diff
@@ -81,25 +87,9 @@ bot.once("spawn", () => {
         console.log(`stop action`);
         bot.pathfinder.setGoal(null);
         break;
-      case "read temperature":
```
Comment (cmendible) [inline minecraftbot/main.ts]: please don remove this case

File: minecraftbot/main.ts [hunk 2]
```diff
@@ -1,9 +1,14 @@
-const axios = require('axios');
```
Comment (cmendible) [inline minecraftbot/main.ts:1]: please do not remove the axios require statement

File: deploy/ops.json [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
         "name": "lordvanmanu",
         "level": 4
     },
+    {
+        "uuid": "2cf95e10-e84c-3e65-b136-c63747c99203",
+        "name": "vicky",
+        "level": 4
+    }
     {
         "uuid": "cmendibl3",
```
Comment (cmendible) [inline deploy/ops.json]: a1a34626-0e38-3153-894c-bcab35ec6bee

**Review by cmendible — CHANGES_REQUESTED** [review]

File: minecraftbot/main.ts (hunk 1 above)
Comment (Manuss20) [reply minecraftbot/main.ts]: solved fix

File: minecraftbot/main.ts (hunk 2 above)
Comment (Manuss20) [reply minecraftbot/main.ts:1]: solved fix

File: deploy/ops.json (hunk 3 above)
Comment (Manuss20) [reply deploy/ops.json]: solved fix

**Review by cmendible — APPROVED** [review]

**Pull request merged**